h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -82,7 +84,7 @@ public static void addModelPredictionsToLevelOneFrame(Model aModel, Frame aModel
     }
   }
 
-  private class StackedEnsembleDriver extends Driver {
```

Comment:
abstracted this driver to provide 2 implementations:
- one for cross-validation stacking strategy.
- one for blending.
StackedEnsembleModel itself is agnostic.